packages/zipkin-instrumentation-redis/test/integrationTest.js

```diff
@@ -73,6 +79,53 @@ describe('redis interceptor', () => {
     });
   });
 
+  it('should add zipkin annotations for batch method', (done) => {
```

Comment:
Is there a test for `multi`? The easiest thing to mess up is the variable scoping of nested functions and wrapping. I'd make couple solid unit tests that verify you can share a the same trace id across multiple nested wrapped command calls. The wrapping and `.apply` can be tricky.

Reply:
I added one (I hope pretty comprehensive test) in cad0528